tests/e2e/molecule/manage_vm/scenarios/update_vm.yml

```diff
@@ -0,0 +1,103 @@
+- name: Import vars
+  ansible.builtin.include_vars:
+    file: ../vars/vars.yml
+
+- name: Provision VM
+  loop:
+    - "{{ provision_vms[0] }}"
+  ansible.builtin.include_tasks:
+    file: manage_vm/tasks/provision_vm.yml
+
+- name: Get info about the VM to be updated
+  ansible.builtin.include_tasks:
+    file: ../tasks/get_vm_info.yml
+  vars:
+    vm_name: "{{ vm_update_name }}"
+
+- name: Set vm_uuid variable
+  ansible.builtin.set_fact:
+    vm_uuid: "{{ vm_info.virtual_machines[0].uuid }}"
+
+- name: Set vm_moid variable
+  ansible.builtin.set_fact:
+    vm_moid: "{{ vm_info.virtual_machines[0].moid }}"
+
+- name: Test updating name of existing VM
+  loop: "{{ vm_names_to_update }}"
+  ansible.builtin.include_tasks:
+    file: manage_vm/tasks/update_vm_name.yml
+
+# - name: Install VMware Tools on VM with ID {{ vm_moid }}
+#   ansible.builtin.include_tasks:
+#     file: provision_vm/tasks/install_vmware_tools.yml
+
+- name: Test changing state of existing VM {{ vm_update_name }}
+  loop: "{{ vm_states }}"
+  ansible.builtin.include_tasks:
+    file: manage_vm/tasks/update_vm_state.yml
+
+- name: Add Management Network VM Portgroup to all hosts in a cluster
+  community.vmware.vmware_portgroup:
+    hostname: "{{ vCenter_hostname }}"
+    username: "{{ vCenter_username }}"
+    password: "{{ vCenter_password }}"
+    validate_certs: "{{ provision_vm_validate_certs }}"
+    cluster_name: "{{ provision_vm_cluster }}"
+    switch: "{{ vswitch_name }}"
+    portgroup: "{{ portgroup_name }}"
+    security:
```

Comment:
these two security settings make the network weaker to attacks. I could be wrong, but I dont think they are needed for a regular vm network

Reply:
thanks, I checked they are indeed not needed. I removed them